rasa/nlu/training_data/formats/rasa_yaml.py

```diff
@@ -322,3 +350,108 @@ def is_yaml_nlu_file(filename: Text) -> bool:
                 f"Error: {e}"
             )
         return False
+
+
+class RasaYAMLWriter(TrainingDataWriter):
+    """Writes training data into a file in a YAML format."""
+
+    def dumps(self, training_data: "TrainingData") -> Text:
+        """Turns TrainingData into a string."""
+        stream = StringIO()
+        self.dump(stream, training_data)
```

Comment:
mhm `stream` isn't a filename (and not a `Text`) as stated in the below signature

Reply:
True! Fixed that.